packages/logger/tests/index.test.ts

```diff
@@ -0,0 +1,20 @@
+import { isClass } from '@sapphire/utilities';
+import { Logger, LoggerLevel, LoggerStyle, LoggerTimestamp } from '../src';
+
+describe('Integration', () => {
+	test('Logger should be a class', () => {
+		expect(isClass(Logger)).toBe(true);
+	});
+
+	test('LoggerLevel should be a class', () => {
+		expect(isClass(LoggerLevel)).toBe(true);
+	});
+
+	test('LoggerStyle should be a class', () => {
+		expect(isClass(LoggerStyle)).toBe(true);
+	});
+
+	test('LoggerTimestamp should be a class', () => {
+		expect(isClass(LoggerTimestamp)).toBe(true);
```

Comment:
nit: These tests are so... redundant and useless... If you want tests, add more :3

Reply:
Added in 3153464.